Content/src/Client/Client.fsproj

```diff
@@ -9,8 +9,7 @@
     <None Include="package.json" />
     <None Include="package-lock.json" />
     <None Include="webpack.config.js" />
-<!--#if (minimal)-->
```

Comment:
please keep those as they are - will be helpful when switching from minimal to full and back while working on the template